index.html

```diff
@@ -10,6 +10,35 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+<header>
+    <section>
+
+        <div class="header">
+
+            <div class="lamda">
```

Comment:
The lambda symbol is part of the nav and not separate.  You can move it into the nav, which may help you when styling the nav with flexbox.